ui/src/cloud/apis/reporting.ts

```diff
@@ -21,7 +21,7 @@ export interface Points {
 export const reportPoints = (points: Points) => {
   try {
     const url = '/api/v2/app-metrics'
-    fetch(url, {
+    return fetch(url, {
```

Comment:
added because it's generally best practice to return promises from functions that return them